.changeset/five-dancers-poke.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/ui-extensions': minor
+'@shopify/ui-extensions-react': minor
```

Comment:
any reason this is a minor change vs a patch?

Reply:
Fixed!